test/clj/rems/test_browser.clj

```diff
@@ -489,15 +508,120 @@
+                (slurp-rows :forms))))))
+
+(defn create-workflow []
+  (testing "create workflow"
+    (btu/with-postmortem {:dir btu/reporting-dir}
+      (go-to-admin-workflows)
+      (btu/scroll-and-click :create-workflow)
+      (btu/wait-visible {:tag :h1 :fn/text "Create workflow"})
+      (select-option "Organization" "nbn")
+      (fill-form-field "Title" (btu/context-get :workflow-name))
+      ;; Default workflow is already checked
+      (select-option "Handlers" "handler")
+      ;; No form
+      (btu/screenshot (io/file btu/reporting-dir "about-to-create-workflow.png"))
+      (btu/scroll-and-click :save)
+      (btu/wait-visible {:tag :h1 :fn/text "Workflow"})
+      (btu/wait-page-loaded)
+      (btu/screenshot (io/file btu/reporting-dir "created-workflow.png"))
+      (is (str/includes? (btu/get-element-text {:css ".alert-success"}) "Success"))
+      (is (= {"Organization" "NBN"
+              "Title" (btu/context-get :workflow-name)
+              "Type" "Default workflow"
+              "Handlers" "Hannah Handler (handler@example.com)"
+              "Active" ""}
+             (slurp-fields :workflow)))
+      (go-to-admin-workflows)
+      (is (some #(= (btu/context-get :workflow-name) (get % "title"))
+                (slurp-rows :workflows))))))
+
+(defn create-catalogue-item []
+  (testing "create catalogue item"
+    (btu/with-postmortem {:dir btu/reporting-dir}
+      (go-to-admin-catalogue)
+      (btu/scroll-and-click :create-catalogue-item)
+      (select-option "Organization" "nbn")
+      (fill-form-field "Name" (str (btu/context-get :catalogue-item-name) " EN") {:index 1})
+      (fill-form-field "Name" (str (btu/context-get :catalogue-item-name) " FI") {:index 2})
+      (fill-form-field "Name" (str (btu/context-get :catalogue-item-name) " SV") {:index 3})
+      (select-option "Workflow" (btu/context-get :workflow-name))
+      (select-option "Resource" (btu/context-get :resid))
+      (select-option "Form" (btu/context-get :form-name))
+      (btu/screenshot (io/file btu/reporting-dir "about-to-create-catalogue-item.png"))
+      (btu/scroll-and-click :save)
+      (btu/wait-visible {:tag :h1 :fn/text "Catalogue item"})
+      (btu/wait-page-loaded)
+      (btu/screenshot (io/file btu/reporting-dir "created-catalogue-item.png"))
+      (is (str/includes? (btu/get-element-text {:css ".alert-success"}) "Success"))
+      (is (= {"Organization" "NBN"
+              "Title (EN)" (str (btu/context-get :catalogue-item-name) " EN")
+              "Title (FI)" (str (btu/context-get :catalogue-item-name) " FI")
+              "Title (SV)" (str (btu/context-get :catalogue-item-name) " SV")
+              "More info (EN)" ""
+              "More info (FI)" ""
+              "More info (SV)" ""
+              "Workflow" (btu/context-get :workflow-name)
+              "Resource" (btu/context-get :resid)
+              "Form" (btu/context-get :form-name)
+              "Active" ""
```

Comment:
Would be nice to add some helper to slurp to load something based on the icons here.

Reply:
maybe once we actually care about the activity status in some test

Reply:
I think the catalogue item could be one.